tutorials/tour/_posts/2017-02-13-implicits.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: tutorial
+title: Implicits
+
+disqus: true
+
+tutorial: scala-tour
+categories: tour
+num: 26
+next-page: implicit-conversions
+previous-page: explicitly-typed-self-references
+---
+Implicits allow for automatic application of code when an explicit application isn't supplied.
+
+## Implicit parameters
+
+Implicit parameters allow for the caller to omit an argument if an implicit one is in scope. Use the `implicit` keyword to make a value, object, or expression implicit. You also use it to make the parameter list implicit.
+```tut
+class Greeting(val greeting: String) {
+  def greet(name: String) = s"$greeting, $name"
+}
+implicit val standardGreeting = new Greeting("Hello")
+
+
+def printGreeting(name: String)(implicit greeting: Greeting) = greeting.greet(name)
+
+
+printGreeting("Franchesca")  // Hello, Franchesca
+printGreeting("Fred")(new Greeting("Good day"))  // Good day, Fred
+```
+The `implicit val standardGreeting` is a value that can be supplied as an argument to an implicit parameter automatically. In the method `printGreeting`, the parameter `greeting` is implicit. This means that the caller can either supply an argument normally or skip it. With `greet("Franchesca")`, the compiler doesn't see a greeting but it notices that `greeting` is an implicit parameter so it searches the current scope for an implicit `Greeting` and finds `standardGreeting`.
+
+This becomes useful when you have a lot of similar arguments to function calls throughout your program. However, implicits can make code more difficult to understand because it's not always obvious where they're defined if you import them from another module with a wildcard (e.g. `import MyPredef._`).
+
+
+
+
+## Implicit conversion
+
+
+An implicit conversion happens the type check fails for an argument and an implicit conversion is found.
```

Comment:
You need to explain what a conversion is: an implicit def that converts a single argument of one type to another type; or an implicit val of a function that does the same thing.

Reply:
So here I need to explain two ways implicit conversions happen:
-An implicit def that converts a single argument of one type to another type
-An implicit val of a function that does the same thing